src/main/java/pl/javastart/task/CoordinateSystem.java

```diff
@@ -4,8 +4,56 @@
 
 public class CoordinateSystem {
 
-    // uzupełnij metodę. Używaj scannera przekazanego w parametrze.
     void run(Scanner scanner) {
-        System.out.println("Punkt (5, -3)");
+        Point point = new Point();
+        System.out.println("Podaj X");
+        point.setX(scanner.nextInt());
+        System.out.println("Podaj Y");
+        point.setY(scanner.nextInt());
+        findPosition(point);
+    }
+
+    void findPosition(Point point) {
+        if (point.getX() != 0 && point.getY() != 0) {
+            showQuarterInfo(point);
+        } else if (point.getX() == 0 && point.getY() == 0) {
+            System.out.println(showPointInfo(point) + " na środku układu współrzędnych");
+        } else {
+            showAxisInfo(point);
+        }
+    }
+
+    private String showPointInfo(Point point) {
```

Comment:
show to wyświetl, a Ty nic w tej metodzie nie wyświetlasz, tylko zwracasz. Lepiej więc pasuje nazwa "getPointInfo". Druga opcja to zmiana tej metody na void i wtedy faktycznie w niej powinieneś mieć System.out.println. Ta druga opcja wydaje się u Ciebie nawet lepsza, bo wtedy metoda findPosition będzie bardziej spójna (teraz albo deleguje wyświetlanie to innej metody, albo sama też ma System.out.printlna)